components/artwork_metadata_stub/templates/contact.jade

```diff
@@ -1,10 +1,16 @@
-if artwork.is_in_auction
-  if artwork.is_biddable
-    .artwork-metadata-stub__contact.artwork-metadata-stub__line.artwork-metadata-stub__bid-now
+if artwork.sale && artwork.sale.is_auction
+  .artwork-metadata-stub__contact.artwork-metadata-stub__line.artwork-metadata-stub__bid-now
+    if artwork.sale.is_live_open
+      a(href= artwork.href) Enter Live Auction
+    else if artwork.sale.is_open
       a(href= artwork.href)
-        | Bid now
```

Comment:
Yup I see that in the new designs, i posted on that issue in case it was an oversight on her part :)